apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/WriteTests.cs

```diff
@@ -150,6 +148,96 @@ public async Task WriteValues()
             });
         }
 
+        [Fact]
+        public async Task WriteValuesDml()
+        {
+            var testTimestamp = new DateTime(2017, 3, 17, 15, 30, 0);
```

Comment:
Rather than copying the data from WriteValues, can we just have some common code?